internal/controller/apiextensions/composite/environment/selector.go

```diff
@@ -90,11 +90,11 @@ func (s *APIEnvironmentSelector) SelectEnvironment(ctx context.Context, cr resou
 		case v1.EnvironmentSourceTypeSelector:
 			ec, err := s.lookUpConfigs(ctx, cr, src.Selector.MatchLabels)
 			if err != nil {
-				return errors.Wrapf(err, errFmtReferenceEnvironmentConfig, i)
+				return errors.Wrapf(err, errFmtReferenceEnvironmentConfig, i, fmt.Sprintln(src.Selector.MatchLabels))
```

Comment:
Should we differentiate this from the below? This is more like "cannot lookup environment config..." rather.